### ISSUE TITLE
Client SDK Demo

### ISSUE THREAD

**Issue by rorygeoghegan** [reservoirprotocol/sample-marketplace#169]
Cancel
When the user tries to do this, we just fetch their orders and cancel the first one. If they have no orders, just say "Create a bid or listing first"

Sell Token
The user needs a token obviously. For this, it's fine to link them out to the paradigm faucet. Same if they have no ETH. Once they have a token, it just needs an offer on it. I think we can do this on-demand. When they try to sell, we call an API which has access to a hot wallet with Rinkeby ETH in it, which instantly creates an offer on their token. They can then accept this Offer

Create Ask / Listing
Similarly, they need to go get a token from Paradigm. Then they can list it

Buy Token
Firstly, they need ETH, which they can get from Paradigm. Then in terms of creating listings for them to buy, I say we just do 10 or so manually, and more if they ever run out. Also, there will be other listings from users testing. And also, if nothing is on sale, it could call an API that checks if the hot wallet has any tokens (due to other testers accepting bids), and quickly lists them all for sale so that the user can buy
so to summarize, every action seems doable, with the only prerequisites being:
1) The user has to go to paradigm to get some ETH or tokens
2) We create orders on the fly via a hot wallet
in my mind, the user is not even able to set the price or parameters. all they do is press one of the buttons. that way, we can always set the price very low, to make sure anyone has enough ETH to fill the orders (e.g. 0.01)
so in the code, there will be a method for each action, which will potentially do some prerequisite checks first (all nice commented), then call the relevant SDK function

https://www.figma.com/file/YBiB8VnOgQSSY73rBobcvS/Reservoir-Marketplace-Builder?node-id=666%3A10166
